### PR TITLE
fix crash: excepted singleton

### PR DESCRIPTION
**Pull request by lepistone**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

File: stock_reserve/model/product.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ class ProductProduct(models.Model):
         compute='_reservation_count',
         string='# Sales')
 
-    @api.multi
+    @api.one
     def _reservation_count(self):
         StockReservation = self.env['stock.reservation']
         product_id = self._ids[0]
```
Comment (yvaucher) [inline stock_reserve/model/product.py]: Could you replace it by self.id ? or use self.id directly in domain ?

**Comment by yvaucher** [comment]
~~Tests are failing~~ just saw #42 

Can you improve the annoted line ?

**Comment by lepistone** [comment]
I tried to add a little innocent test, and :snake: :snake: came out. I'm fixing this. Don't merge, please.

**Comment by lepistone** [comment]
We should be good now!

**Comment by jgrandguillaume** [comment]
:+1:

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**